bedrock/firefox/redirects.py

```diff
@@ -24,9 +24,12 @@ def firefox_mobile_faq(request, *args, **kwargs):
     }),
 
     # bug 748503
-    redirect(r'^projects/firefox/[^/]+a[0-9]+/(?:firstrun|whatsnew)(?P<p>.*)$',
+    redirect(r'^projects/firefox/[^/]+a[0-9]+/firstrun(?P<p>.*)$',
```

Comment:
NM. I think I see what's up. The real view will just handle this now? But I think we shouldn't show the new page for every versioned URL right? We should redirect to the current URL?